db/handle_buf.c

```diff
@@ -1015,9 +1015,9 @@ int handle_buf_main2(struct dbenv *dbenv, SBUF2 *sb, const uint8_t *p_buf,
                     if (add_latency > 0) {
                         poll(0, 0, rand() % add_latency);
                     }
-                    time_metric_add(thedb->handle_buf_queue_time,
-                                    comdb2_time_epochms() -
-                                        nextrq->queue_time_ms);
+                    time_metric_add(thedb->handle_buf_queue_time, comdb2_time_epochms() - nextrq->queue_time_ms);
+                    time_metric_add(thedb->queue_depth,
+                                    q_reqs.count + thdpool_get_queue_depth(get_default_sql_pool(0)));
```

Comment:
`handle_buf` queue size isn't actively accounted for. It's added to the averager in [dispatch_sql_query](https://github.com/bloomberg/comdb2/blob/master/db/sqlinterfaces.c#L5370), which means that if there's no SQL activity on the master node, the master's handle_buf queue size won't be reported. The patch fixes it here.